src/components/Sqlninja/sqlninja.tsx

```diff
@@ -0,0 +1,225 @@
+import { Button, NativeSelect, Stack, TextInput, Switch } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState, useEffect } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```

Comment:
You can use LoadingOverlayAndCancelButtonPkexec 

```suggestion
import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```